### DISCUSSION TITLE
How exactly do you uninstall on Linux?

### DISCUSSION THREAD

**Discussion by GiuffreLab** [JanDeDobbeleer/oh-my-posh#1653] [Troubleshoot]
Cant seem to find any information on how to uninstall / remove oh my posh on a linux distro. Any information on how to do this?

**Comment by GiuffreLab** [comment]
Unsure if this was correct, but I removed oh-my-posh from the /usr/local/bin folder, removed the .poshthemes folder and removed the line from .bashrc

**Reply by JanDeDobbeleer** [reply]
That's correct!
